src/components/views/rooms/EventTile.tsx

```diff
@@ -619,11 +619,8 @@ export default class EventTile extends React.Component<IProps, IState> {
             <div
                 className="mx_ThreadInfo"
                 onClick={() => {
-                    dispatchShowThreadEvent(
-                        this.props.mxEvent,
-                    );
-                }}
-            >
+                    dispatchShowThreadEvent(this.props.mxEvent);
+                }}>
```

Comment:
This seems unrelated, should it be reverted...?

Reply:
I prefer how it looks with those changes. But you are right that it does not fit into this pr. I guess I should revert it.